code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -234,7 +234,7 @@
 			if(M_ELVIS in mutations)
 				src.emote("fart")
 				msg = "<B>[src]</B> has left the building..."
-			if(M_HARDCORE in mutations)
+			if(M_HARDCORE in mutations && !miming)
```

Comment:
I tested it and yes, it does. Oops.

I can just swap them around and it'll work fine, right?